internal/job/checkout.go

```diff
@@ -780,12 +781,21 @@ func gitFetchCommitWithFallback(ctx context.Context, shell *shell.Shell, gitFetc
 
 const CommitMetadataKey = "buildkite:git:commit"
 
+var potentiallyValidGitSHA = regexp.MustCompile(`^[a-f0-9]{40}|[a-f0-9]{64}$`)
```

Comment:
If I suggest my usual hobby-horse of naming regexps ending in `RE`, then we get:

```go
var potentiallyValidGitSHARE = regexp.MustCompile(`^[a-f0-9]{40}|[a-f0-9]{64}$`)
```

Hmm 🤔  What about:

```suggestion
var potentiallyValidGitHashRE = regexp.MustCompile(`^[a-f0-9]{40}|[a-f0-9]{64}$`)
```

Reply:
Makes sense to match the naming, have updated from your suggestion!